.github/workflows/tests.yaml

```diff
@@ -553,6 +554,7 @@ jobs:
                   echo "FABRIC_BRIDGE_APP: out/linux-x64-fabric-bridge-rpc-${BUILD_VARIANT}-clang/fabric-bridge-app" >> /tmp/test_env.yaml
                   echo "FABRIC_SYNC_APP: out/linux-x64-fabric-sync-${BUILD_VARIANT}-clang/fabric-sync" >> /tmp/test_env.yaml
                   echo "LIGHTING_APP_NO_UNIQUE_ID: out/linux-x64-light-data-model-no-unique-id-${BUILD_VARIANT}-clang/chip-lighting-app" >> /tmp/test_env.yaml
+                  echo "CLOSURE_APP: out/linux-x64-closure/chip-closure-app" >> /tmp/test_env.yaml
```

Comment:
This changes is not sufficient to run the closure control or dimension tests using the closure app. There are extra changes necessary for the app to be used in `scripts/tests/chiptest` scripts and lower in this yaml.